src/user-management/user-management.spec.ts

```diff
@@ -29,8 +30,25 @@ const magicAuthId = 'magic_auth_01H5JQDV7R7ATEYZDEG0W5PRYS';
 const passwordResetId = 'password_reset_01H5JQDV7R7ATEYZDEG0W5PRYS';
 
 describe('UserManagement', () => {
+  const originalEnv = process.env;
+  let workos: WorkOS;
+
+  beforeAll(() => {
+    jest.resetModules();
+    process.env = {
+      ...originalEnv,
+      WORKOS_CLIENT_ID: 'proj_123',
+    };
+
+    workos = new WorkOS('sk_test_Sz3IQjepeSWaI4cMS4ms4sMuU');
```

Comment:
For some reason when under test the `baseURL` remains undefined, even though the constructor is called. This is causing all the UM tests to fail since we use the baseURL to set up the JWKs. Any ideas why that might be?